Packs/CommonScripts/Scripts/ShowScheduledEntries/ShowScheduledEntries.js

```diff
@@ -14,7 +14,9 @@ md += '-|-\n';
 
 var scheduledEntries = [];
 entries.forEach(function (entry) {
-    if (entry.Metadata.Recurrent && entry.Metadata.Schedule.Scheduled) {
+    logDebug(entry);
```

Comment:
please log it if the condition does not hold, for future debugging (rather than logging every entry)